Source/Microsoft.Teams.Apps.FAQPlusPlus/Bots/FaqPlusPlusBot.cs

```diff
@@ -788,10 +797,10 @@ private static Task<TaskModuleResponse> GetTaskModuleResponseAsync(Attachment qu
                     break;
 
                 case Constants.ShareFeedback:
-                    if (string.IsNullOrEmpty(await this.configurationProvider.GetSavedEntityDetailAsync(ConfigurationEntityTypes.FeedbackTeamId)) == true)
+                    if (string.IsNullOrEmpty(await this.configurationProvider.GetSavedEntityDetailAsync(ConfigurationEntityTypes.FeedbackTeamId).ConfigureAwait(false)))
                     {
                         this.logger.LogInformation("Feedback Team ID not registered yet.");
-                        await turnContext.SendActivityAsync(MessageFactory.Text("I'm sorry, feedback team has not been registered yet.", "I'm sorry, feedback team has not been registered yet."), cancellationToken).ConfigureAwait(false);
+                        await turnContext.SendActivityAsync(MessageFactory.Text(Strings.FeedbackTeamUnregisteredMessage, Strings.FeedbackTeamUnregisteredMessage), cancellationToken).ConfigureAwait(false);
```

Comment:
I'm not sure you need to use two strings when you use the turnContext.SendActivityAsync() method. Can you double check?

Reply:
The second string is optional, I think that was how it was followed in the echo bot example. I have it edited out since it won't be required.

Reply:
That's good!